mono_repo/README.md

```diff
@@ -114,6 +114,15 @@ github:
           env:
             CHAT_WEBHOOK_URL: ${{ secrets.CHAT_WEBHOOK_URL }}
 
+  # You can customize stage ordering as well as make certain stages be
+  # conditional here, this is supported for all CI providers. The `if`
+  # condition should use the appropriate syntax for the provider it is being
+  # configured for.
+  stages:
+    - name: cron
+      # Only run this stage for scheduled cron jobs
+      if: github.event_name == 'schedule'
```

Comment:
This is only for github, each CI provider configures this separately because they have different syntax.

I could also show an example for travis in the readme, but I think it would be duplicative?

Reply:
so...uh...why is this setting not under the `github` key?

Reply:
It is

Reply:
Oh. 🤦